actors/builtin/miner/miner_test.go

```diff
@@ -2778,6 +2892,30 @@ func (h *actorHarness) changeWorkerAddress(rt *mock.Runtime, newWorker addr.Addr
 
 }
 
+func (h *actorHarness) checkSectorProven(rt *mock.Runtime, sectorNum abi.SectorNumber) {
+	param := &miner.CheckSectorProvenParams{sectorNum}
+
+	rt.ExpectValidateCallerAny()
+
+	rt.Call(h.a.CheckSectorProven, param)
+	rt.Verify()
+}
+
+func (h *actorHarness) changeMultiAddrs(rt *mock.Runtime, newAddrs []abi.Multiaddrs) {
```

Comment:
I think I conflicted with you here a bit. I've added some tests for multiaddrs/peer ids as part of #920.

Reply:
@Stebalien Never mind. I'll rebase after your PR goes in.